packages/devtools/lib/src/main.dart

```diff
@@ -128,8 +130,75 @@ class PerfToolFramework extends Framework {
     showError(title, error);
   }
 
-  void initTestingModel() {
-    App.register(this);
+  Future<bool> isFlutterApp() async {
+    final EvalOnDartLibrary flutterLibrary = EvalOnDartLibrary(
+      flutterLibraryUriCandidates,
+      serviceManager.service,
+    );
+
+    try {
+      await flutterLibrary.libraryRef.catchError(
```

Comment:
why isn't this
try {
  await flutterLibrary.libraryRef;
} catch LibraryNotFound (e) {
}
?